builtin/mainmenu/pkgmgr.lua

```diff
@@ -100,7 +100,15 @@ local function load_texture_packs(txtpath, retval)
 	end
 end
 
-function get_mods(path, virtual_path, retval, modpack)
+--modmanager implementation
+pkgmgr = {}
+
+--- Scans a directory recursively for mods and adds them to `listing`
+-- @param path         Absolute directory path to scan recursively
+-- @param virtual_path Prettified unique path (e.g. "mods", "mods/mt_modpack")
+-- @param listing      Input. Flat array to insert located mods and modpacks
```

Comment:
Shouldn't "Input" be "Output"?

Reply:
Yes and no. It's a mandatory table input which is then modified by reference